heidrun/ia.rb

```diff
@@ -0,0 +1,238 @@
+# coding: utf-8
+
+# TODO: this list is from the old system and is likely out of date
```

Comment:
I thought it was out of date because one of the collections I tried didn't seem to be there.